### PR TITLE
removing ldas remnants

### PR DESCRIPTION
**Pull request by ahnitz**
Replace hard-coded ldas coefficients with the scipy firwin equivelant.

### CODE REVIEW

**Comment by ahnitz** [comment]
Test workflow will appear here. https://www.atlas.aei.uni-hannover.de/~ahnitz/LSC/ahnitz/projects/code_testing/q1/resc/html

**Comment by titodalcanton** [comment]
I'm curious how close the LDAS coefficients are to what you get via scipy... Did you check that?

**Comment by titodalcanton** [comment]
Also, it may be worth putting a comment describing why we choose a Kaiser window for future reference. Even just "we use what LDAS used" would do.

**Comment by ahnitz** [comment]
Ok, I had spot checked the coefficients before, but here is a script to do so. 

import numpy, scipy.signal

ldas = numpy.array( \
    [-7.1587345178999983e-19, -1.0514587726686521e-03, 1.8542385840153159e-18,
    2.5089668121365313e-03, -3.4940298359108189e-18, -4.8948343392593557e-03,
    5.6062660030794468e-18, 8.5565590024681924e-03, -8.0907458797357440e-18,
    -1.3989973238439804e-02, 1.0780775955476128e-17, 2.2023120574050724e-02,
    -1.3459232902020823e-17, -3.4340179881765416e-02, 1.5884076076327108e-17,
    5.5288283911880384e-02, -1.7819618555760906e-17, -1.0093019739775573e-01,
    1.9068667053692095e-17, 3.1670034568032440e-01, 5.0025873529805742e-01,
    3.1670034568032440e-01, 1.9068667053692095e-17, -1.0093019739775573e-01,
    -1.7819618555760906e-17, 5.5288283911880384e-02, 1.5884076076327108e-17,
    -3.4340179881765416e-02, -1.3459232902020823e-17, 2.2023120574050724e-02,
    1.0780775955476128e-17, -1.3989973238439804e-02, -8.0907458797357440e-18,
    8.5565590024681924e-03, 5.6062660030794468e-18, -4.8948343392593557e-03,
    -3.4940298359108189e-18, 2.5089668121365313e-03, 1.8542385840153159e-18,
    -1.0514587726686521e-03, -7.1587345178999983e-19])

sci = scipy.signal.firwin(41, .5, window=('kaiser', 5))

rel = (ldas - sci) / sci

ab = abs(ldas - sci)

print rel, ab

[ -3.30261134e-05  -6.15281791e-12  -3.30261136e-05  -4.45734416e-12
  -3.30261144e-05  -5.76447388e-12  -3.30261140e-05  -1.54424094e-12
  -3.30261136e-05   3.32685632e-13  -3.30261143e-05  -1.08479684e-12
  -3.30261166e-05   5.96895318e-13  -3.30261136e-05   4.22571549e-13
  -3.30261138e-05   2.16698215e-13  -3.30261140e-05   7.46691652e-14
   5.45947216e-14   7.46691652e-14  -3.30261140e-05   2.16698215e-13
  -3.30261138e-05   4.22571549e-13  -3.30261136e-05   5.96895318e-13
  -3.30261166e-05  -1.08479684e-12  -3.30261143e-05   3.32685632e-13
  -3.30261136e-05  -1.54424094e-12  -3.30261140e-05  -5.76447388e-12
  -3.30261144e-05  -4.45734416e-12  -3.30261136e-05  -6.15281791e-12
  -3.30261134e-05] [  2.36432986e-23   6.46943436e-15   6.12403166e-23   1.11833286e-14
   1.15398040e-22   2.82161447e-14   1.85159295e-22   1.32133887e-14
   2.67214717e-22   4.65426309e-15   3.56058898e-22   2.38906117e-14
   4.44520877e-22   2.04974926e-14   5.24606627e-22   2.33632558e-14
   5.88532188e-22   2.18713936e-14   6.29784772e-22   2.36477504e-14
   2.73114864e-14   2.36477504e-14   6.29784772e-22   2.18713936e-14
   5.88532188e-22   2.33632558e-14   5.24606627e-22   2.04974926e-14
   4.44520877e-22   2.38906117e-14   3.56058898e-22   4.65426309e-15
   2.67214717e-22   1.32133887e-14   1.85159295e-22   2.82161447e-14
   1.15398040e-22   1.11833286e-14   6.12403166e-23   6.46943436e-15
   2.36432986e-23]

Bottom line. When the absolute differences are average, the relative differences are at double precision level. When the absolute differences are orders of magnitude smaller, the relative differences is about 1e-5, which is consistent with a numerical accuracy loss. They are functionally identical as far as I can tell as filters.

**Comment by titodalcanton** [comment]
Ok, the test workflow looks sane and the filters are equivalent.
